appstore/validator.go

```diff
@@ -81,6 +82,7 @@ func New() Client {
 	client := Client{
 		URL:     SandboxURL,
 		TimeOut: time.Second * 5,
+		SandboxURL: SandboxURL,
 	}
 	if os.Getenv("IAP_ENVIRONMENT") == "production" {
 		client.URL = ProductionURL
```

Comment:
If we always need to hit the production server first, do we still need to keep this line? It seems regardless of the environment we just need to hit production -> sandbox in order.

Reply:
For the same reason, I think we can eliminate Config.IsProduction

Reply:
Thanks for the suggestion! Removed this line and set it to the productionURL by default. Also got rid of the isProduction config since its not good practice to hit the sandbox directly anyways.